src/penn_chime/parameters.py

```diff
@@ -64,30 +65,36 @@ def __init__(
         recovered: int = 0,
         region: Optional[Regions] = None,
     ):
-        self.current_hospitalized = current_hospitalized
-        self.relative_contact_rate = relative_contact_rate
+        self.current_hospitalized = StrictlyPositive(value=current_hospitalized)
+        self.relative_contact_rate = Rate(value=relative_contact_rate)
+
+        Rate(value=hospitalized.rate), Rate(value=icu.rate), Rate(value=ventilated.rate)
+        StrictlyPositive(value=hospitalized.days), StrictlyPositive(value=icu.days),
+        StrictlyPositive(value=ventilated.days)
 
         self.hospitalized = hospitalized
         self.icu = icu
         self.ventilated = ventilated
 
         if region is not None and population is None:
             self.region = region
-            self.population = region.population
+            self.population = StrictlyPositive(value=region.population)
         elif population is not None:
             self.region = None
-            self.population = population
+            self.population = StrictlyPositive(value=population)
         else:
             raise AssertionError('population or regions must be provided.')
 
         self.current_date = current_date
+       
         self.date_first_hospitalized = date_first_hospitalized
         self.doubling_time = doubling_time
```

Comment:
If `doubling_time is not None` then it should be `StrictlyPositive` too.

Reply:
Ah, that would be your just added `OptionalBounded`. Excellent.

Reply:
doublign time is validated w `OptionalStrictlyPositive`, lack of upper bound was verified by Corey on slack